src/tests/suite/daos_rebuild_common.c

```diff
@@ -79,13 +81,14 @@ rebuild_extend_tgt(test_arg_t **args, int args_cnt, d_rank_t rank,
 		   int tgt_idx, daos_size_t nvme_size)
 {
 	int i;
+	int rc = 0;
 
 	for (i = 0; i < args_cnt; i++) {
-		if (!args[i]->pool.destroyed)
-			daos_extend_target(args[i]->pool.pool_uuid,
-					   args[i]->group,
-					   args[i]->dmg_config,
-					   rank, tgt_idx, nvme_size);
+		if (!args[i]->pool.destroyed) {
+			rc = dmg_pool_extend(args[i]->dmg_config, args[i]->pool.pool_uuid,
```

Comment:
I see tgt_idx is now not passed (since dmg pool extend doesn't accept --target-idx argument?)
But I guess the tgt_idx argument to this function is now unused and could be removed here and by the callers?

Reply:
Sounds good to defer on this for now